README.md

```diff
@@ -16,14 +16,14 @@ docker-emulator-android is one of the components of [android-farm](https://githu
 # Usage
 For example to run default emulator options with Marshmallow (API 23):
 ```console
-$ docker run --rm --privileged -e ANDROID_ARCH="x86" -v /dev/kvm:/dev/kvm agoda-com/docker-emulator-android-23:latest
+$ docker run --rm --privileged -e ANDROID_ARCH="x86_64" -v /dev/kvm:/dev/kvm agoda-com/docker-emulator-android-23:latest
```

Comment:
x86 is significantly faster than x86_64. Please provide benchmarks if that's not the case anymore

Reply:
With the recent image, I haven't found that x86 is any faster than x86_64. But that is true for our current setup.